lib/fhir_client/client.rb

```diff
@@ -37,14 +38,14 @@ class Client
     # @param default_format Default Format Mime type
     # @return
     #
-    def initialize(base_service_url, default_format: FHIR::Formats::ResourceFormat::RESOURCE_XML, proxy: nil)
+    def initialize(base_service_url, default_format: FHIR::Formats::ResourceFormat::RESOURCE_JSON, proxy: nil)
```

Comment:
This would be a breaking change for XML users.  I'm ok with it as long as it is intentional and doesn't mess up any of our examples.

Reply:
Changing the default to json was intentional on my part.  It's a small change that I can roll back if desired, but I think it's a sensible default.

I updated the README to explicitly mention the json default and how to switch between json and xml.

Reply:
Ok.  I do worry we might be in a bubble regarding actual JSON vs. XML usage, but I do think that JSON fits better / is more popular with RESTful interfaces, so defaulting to JSON is fine by me.  @jawalonoski any objections?

Reply:
The current published [Argonaut Data Query Implementation Guide](https://www.fhir.org/guides/argonaut/r2/Conformance-server.html) requires JSON, whereas XML is only a should support.  The [ONC Draft Standardized API Test Procedure](https://www.healthit.gov/sites/default/files/page/2019-03/170_315g_10_Standardized_API_for_Patient_and_Population_Services.pdf) only looks for and requires JSON as well.

I know you know this, but it might be good information for any poor soul who finds they need to update so that they may consider moving to JSON instead of just adding `default_xml`.

Reply:
When I said bubble, the Argo DQ IG / ONC Cures was the bubble I was referring to :) There are a lot of other IGs and use cases under development, and I just wonder if the broad trend is to move to JSON or if we just happen to be focusing on a use case right now that prefers JSON (app-based integrations).

Reply:
I'm going to keep this change.